.changeset/sweet-feet-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-linguist-backend': patch
+---
+
+Breaking change for the alpha export plugin options from exported `linguistPlugin()` in favour of static config.
```

Comment:
I think we still want to have this prefixed with `**BREAKING**:`. Also we should include mention that this is only used by the new backend system. Maybe like this:
```suggestion
**BREAKING**: Removed the alpha export plugin options from exported `linguistPlugin()` (used by the new backend system) in favour of static config.
```

Reply:
sure - happy with it. still keep it as a `patch` though? :)

Reply:
Good question, I think so as it's not a `1.0.0` version yet

Reply:
ok - updated to `minor`